pkg/controllers/analytics.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright © 2021 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package controllers
+
+import (
+	"github.com/DopplerHQ/cli/pkg/http"
+)
+
+// This package collects anonymous analytics for the purpose of improving the Doppler CLI
+
+func CaptureCommand(command string) {
+	// wait for analytics command to complete before exiting
+	ch := make(chan bool, 1)
```

Comment:
What's happening with this channel here? Won't this function just block until `CaptureCommand` finishes?

Reply:
https://pkg.go.dev/sync#WaitGroup Might be a good feature to leverage. 
`wg.Wait()` could happen as the last line of the `Execute()` function in `root.go` and allow wg to be accessed by goroutines in our code.
The first line of the `CaptureCommand` function could be `wg.Add()` and the first line inside the goroutine function `defer wg.Done()`

We can then get rid of the channel. This will ensure that the `Execute()` function does not end (until the wait group counter is done to 0) which would end the main thread and end all goroutines.

Reply:
This indeed was not working as I had intended. I've switched to using a WaitGroup for this.